src/classes/Callable_API.cls

```diff
@@ -105,6 +105,19 @@ global with sharing class Callable_API implements System.Callable {
                 RD2_ApiService rd2ApiSvc = new RD2_ApiService();
                 return rd2ApiSvc.querySchedules(rdIds);
 
+            } when 'rd2.queryinstallments' {
+                Set<Id> rdIds = paramService
+                    .withRequiredParamCheck()
+                    .getSetId(CallableApiParameters.PARAM_RECORD_IDS);
+                Date startDate = paramService
+                    .withoutRequiredParamCheck()
```

Comment:
Suggestion: since required attribute is `false` by default, not need to explicitly check `withoutRequiredParamCheck()`.